setup.py

```diff
@@ -46,6 +46,7 @@ def _load_py_module(fname, pkg="pytorch_lightning"):
     "strategies": setup_tools._load_requirements(path_dir=_PATH_REQUIRE, file_name="strategies.txt"),
     "test": setup_tools._load_requirements(path_dir=_PATH_REQUIRE, file_name="test.txt"),
 }
+extras.update(setup_tools._separate_requirements(extras["strategies"]))
```

Comment:
Observe this magic!

```suggestion
from pkg_resources import parse_requirements

extras.update({req.key: str(req) for req in parse_requirements(os.path.join(_PATH_REQUIRE, "strategies.txt"))})
```

You will also need to fix some things, but you get the idea. We should try to use as much code from the stdlib as possible

Reply:
wow, nice, did not know this function :rabbit: